xla/stream_executor/gpu/gpu_timer.cc

```diff
@@ -124,9 +130,13 @@ DeviceMemory<GpuSemaphoreState> GpuTimer::GpuSemaphore::device() {
   if (ShouldLaunchDelayKernel()) {
     // Check the assumption that this device supports unified addressing,
     // otherwise skip the delay kernel
+#if GOOGLE_CUDA
+    auto attr = CU_DEVICE_ATTRIBUTE_UNIFIED_ADDRESSING;
+#elif TENSORFLOW_USE_ROCM
+    auto attr = hipDeviceAttributeUnifiedAddressing;
+#endif
```

Comment:
I suggest to at least move this out to a function so that we don't have inline ifdefs in the code.

Reply:
looks like gpu_timer has been rolled back again, hence this is not relevant for now..